XenModel/Actions/VM/ImportVmAction.cs

```diff
@@ -422,6 +316,9 @@ public void EndWizard(bool start, List<VIF> vifs)
 			m_startAutomatically = start;
 			m_VIFs = vifs;
 
+            if (m_startAutomatically)
+                ApiMethodsToRoleCheck.Add("vm.start");

```

Comment:
You're right. I also think it would be better if we launched the VMStartAction here instead of calling directly VM.start. I have done this for CA-64500, so if that's ok, I'll leave this fix out of this PR and include it to the upcoming one.